e2e_tests/tests/deploy/test_local.py

```diff
@@ -114,6 +114,24 @@ def test_cluster_down() -> None:
     assert len(containers) == 0
 
 
+@pytest.mark.det_deploy_local
+def test_ee_cluster_up() -> None:
+    name = "test_ee_cluster_up"
+
+    with resource_manager(Resource.CLUSTER, name, {}, ["no-gpu", "enterprise-edition"]):
+        container_name = name + "_determined-master_1"
+        client = docker.from_env()
+
+        containers = client.containers.list(filters={"name": container_name})
+        assert len(containers) > 0
+
+        for container in containers:
+            if "Config" in container.attrs and "Image" in container.attrs["Config"]:
+                assert "hpe-mlde-master" in container.attrs["Config"]["Image"]
+    containers = client.containers.list(filters={"name": container_name})
```

Comment:
I prefer this portion not be included in the test in order to make the point of the test as clear as possible. If you prefer it left because you think that it's important to test bringing EE down separate from tests about bringing OSS down, I can be swayed and let's chat.

Reply:
Ah. I see you went the removal route in the other additions. In that case, I think this assertion is waiting to be removed.

Reply:
oh sorry I missed this one